File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.py
def solution(a, b):
    # a ⊕ b를 문자열로 만들어 정수로 변환
    ab = int(str(a) + str(b))
    
    # 2 * a * b 계산
    product = 2 * a * b
    
    # 두 값 중 더 큰 값을 반환, 같으면 a ⊕ b를 반환
    return ab if ab >= product else product

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.swift
import Foundation

func solution(_ a: Int, _ b: Int) -> Int {
    // a ⊕ b를 문자열로 만들어 정수로 변환
    let ab = Int("\(a)\(b)")!
    
    // 2 * a * b 계산
    let product = 2 * a * b
    
    // 두 값 중 더 큰 값을 반환, 같으면 a ⊕ b를 반환
    return ab >= product ? ab : product
}